packages/@o3r/application/schemas/cms.schema.json

```
@@ -0,0 +1,35 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "CmsMetadataSchema",
+  "type": "object",
+  "title": "Schema of CMS application configuration",
+  "description": "Schema used to indicate the configuration to the CMS when the application is loaded by it",
+  "additionalProperties": false,
+  "properties": {
+    "assetsFolder": {
+      "type": "string",
+      "description": "The relative path to the assets folder to import",
+      "default": "dist/assets"
+    },
+    "localisationsFolder": {
+      "type": "string",
+      "description": "The relative path to the localisations (.json files) folder to import"
+    },
+    "placeholderCssFiles": {
+      "type": "array",
+      "description": "An array of the CSS files which are used by placeholders at runtime. These files will be used in the placeholder editor",
+      "items": {
+        "type": "string"
+      }
+    },
+    "defaultLanguage": {
+      "type": "string",
+      "description": "The language code present in the localistation.metadata.json file",
```

```suggestion
      "description": "The language code present in the localization.metadata.json file",
```

(if you don't want to replace the s by z, the typo with the t should still be replaced: 
`localisation.metadata.json`

---

Actually we can not replace the `z` because of the agreement on the name with CMS